dispatcher/dispatcher.py

```diff
@@ -20,75 +21,18 @@ def __init__(self):
     def interrupt(self, *args):
         self.event.set()
 
-class LogHandler(logging.Handler):
-    def __init__(self, logdir='/live_data/redax_logs/', retention=0):
-        logging.Handler.__init__(self)
-        self.today = datetime.date.today()
-        self.logdir = logdir
-        self.retention = retention
-        self.Rotate(self.today)
-        self.count = 0
-
-    def close(self):
-        if not self.f.closed:
-            self.f.flush()
-            self.f.close()
-
-    def __del__(self):
-        self.close()
-
-    def emit(self, record):
-        msg_today = datetime.date.fromtimestamp(record.created)
-        msg_datetime = datetime.datetime.fromtimestamp(record.created)
-        if msg_today != self.today:
-            self.Rotate(msg_today)
-        m = self.FormattedMessage(msg_datetime, record.levelname, record.msg)
-        self.f.write(m)
-        print(m[:-1]) # strip \n
-        self.count += 1
-        if self.count > 2:
-            self.f.flush()
-            self.count = 0
-
-    def Rotate(self, when):
-        if hasattr(self, 'f'):
-            self.f.close()
-        self.f = open(self.FullFilename(when), 'a')
-        if self.retention == 0:
-            return
-        last_file = when - datetime.timedelta(days=self.retention)
-        if os.path.exists(self.FullFilename(last_file)):
-            os.remove(self.FullFilename(last_file))
-            m=self.FormattedMessage(datetime.datetime.utcnow(), "init", "Deleting " + self.Filename(last_file))
-        else:
-            m=self.FormattedMessage(datetime.datetime.utcnow(), "init", "No older file to delete :(")
-        self.f.write(m)
-        self.today = datetime.date.today()
-
-    def FullFilename(self, when):
-        return os.path.join(self.logdir, self.Filename(when))
-
-    def Filename(self, when):
-        return f"{when.year:04d}{when.month:02d}{when.day:02d}_dispatcher.log"
-
-    def FormattedMessage(self, when, level, msg):
-        return f"{when.isoformat()} | [{str(level).upper()}] | {msg}\n"
-
-
 def main():
 
     # Parse command line
     parser = argparse.ArgumentParser(description='Manage the DAQ')
     parser.add_argument('--config', type=str, help='Path to your configuration file',
-            default='config.ini')
+            default='config_test.ini')
     parser.add_argument('--log', type=str, help='Logging level', default='DEBUG',
             choices=['DEBUG','INFO','WARNING','ERROR','CRITICAL'])
     args = parser.parse_args()
     config = configparser.ConfigParser()
     config.read(args.config)
-    logger = logging.getLogger('main')
-    logger.addHandler(LogHandler())
-    logger.setLevel(getattr(logging, args.log))
+    logger = get_daq_logger('dispatcher', level = getattr(logging, args.log))
```

Comment:
This is the production logger identifier, `dispatcher_test` would be better

Reply:
Let's do this nicely and get it from the config

Reply:
What do you mean Joran? I don't see where it can be written in the config file.

Reply:
See, for instance, https://github.com/AxFoundation/redax/pull/23/files#diff-034fd074be98c324f39e29ede05ef73159b7376a20fcf312549f0359f07e7a93R4

Reply:
I see, I did change it. Not sure if we also need the `logging` in `getattr()` and by now I delete it

Reply:
That depends if https://github.com/XENONnT/daqnt/pull/24 has been deployed (I want to say not yet? Should fix)

Reply:
Ok, so shall I come back to the version of the function gettattr() with the logging by now?
I will write this: `logger = get_daq_logger(config['DEFAULT']['LogName'], level = getattr(logging, args.log))`
I was not sure as I don't know if these two kinds of logger exclude each other. In this PR #23 it is removed.